### PR TITLE
two-line script

### PR DESCRIPTION
**Pull request by brgcode**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by gonzalocasas** [comment]
I'm a tad confused with the various PRs on this, but if you want to re-add cython and planarity on the conda requirements, the way to do it -I think- would be with:
```
requirements:
// here all the other stuff already in there
    - cython
    - planarity       # [not win]
```

**Comment by brgcode** [comment]
i know, but i am trying to fix the problem with matplotlib

**Comment by brgcode** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -5,32 +5,36 @@
...
 
 source:
-  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  fn: {{ name }}-{{ version }}.{{ file_ext }}
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
-  '{{ hash_type }}': '{{ hash_value }}'
+  {{ hash_type }} : {{ hash_value }}
 
 build:
+  skip: True   # [win]
```
Comment (gonzalocasas) [inline recipe/meta.yaml]: These two will skip the entire platform.

Comment (brgcode) [reply recipe/meta.yaml]: i know but those tests have already passed. will remove the flag after i figure out what does the trick on linux...

Comment (gonzalocasas) [reply recipe/meta.yaml]: ah! oki! 👍

Comment (brgcode) [reply recipe/meta.yaml]: do you know why the appveyor stuff sometimes fails because of an SSH certificate?

Comment (gonzalocasas) [reply recipe/meta.yaml]: no clue, looks like a problem on their end...

**Pull request merged**